src/widgets/dialogs/UserInfoPopup.cpp

```diff
@@ -355,6 +356,10 @@ UserInfoPopup::UserInfoPopup(bool closeAutomatically, Split *split)
 
                 this->ui_.nameLabel = addCopyableLabel(box, "Copy name");
                 this->ui_.nameLabel->setFontStyle(FontStyle::UiMediumBold);
+                this->ui_.nameLabel->setPadding(QMargins(8, 0, 0, 0));
+                this->ui_.liveIndicator = new LiveIndicator(8);
```

Comment:
I'd prefer if the LiveIndicator is shown if the stream is live, and hidden if the stream is offline. Not used as padding.
It would mean modifying the `nameLabel` padding on the fly if you want it to look like it does now, but that seems trivial with your modifications to `Label::setPadding`